src/python/deepgnn/graph_engine/snark/tests/util_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Unit test utils."""
```

Comment:
Can we use these mocks as fixtures instead and let pytest to cache them?

Reply:
These are just functions, so not directly. Since they are used only within fixtures though they are cached.